app/assets/javascripts/global.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(function() {
+    var copyTextBtn = $('.copy-text-btn');
+
+    $(copyTextBtn).on('click', function() {
+        var copyTextarea = $('.copy-text');
+        copyTextarea.focus();
+        copyTextarea.select();
+        try {
+            var successful = document.execCommand('copy');
+            var msg = successful ? 'successful' : 'unsuccessful';
+            console.log('Copying text command was ' + msg);
```

Comment:
Should we provide any feedback to the user about whether it was successful or not?